ui/.env

```diff
@@ -1,3 +1,4 @@
 KEYCLOAK_CLIENT_ID="dreams"
 KEYCLOAK_REALM=plato
-DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
\ No newline at end of file
+DATABASE_URL='postgres://postgres:postgres@localhost:35432/cobudget'
+HELPSCOUT_KEY=e1d18cac-eb2f-43c0-a109-6b185575b27e
```

Comment:
I think it's better to remove this from here and supply it in vercel instead. i don't think we want other people deploying the app to send their users to our support.

I just added it to vercel now so it should be safe to remove from here.

also check that the app still work if the variable isn't supplied

Reply:
If HELPSCOUT_KEY is missing, then the Helpscout Beacon is not shown. So if i remove it locally, Beacon will not be shown on the page. The app will still be working.

Reply:
yep that's good but i think we want to remove the line from this file